regression/verilog/synthesis/continuous_assignment_to_net_part_select1.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+continuous_assignment_to_net_part_select1.sv
+--bound 0
+^\[main\.p0\] always main\.some_wire\[1:0\] == 1: PROVED up to bound 0$
+^EXIT=0$
+^SIGNAL=0$
+--
```

Comment:
Perhaps include a link to the issue (or some other description) after another `--` line?